recipes/pydrobert-kaldi/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "pydrobert-kaldi" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydrobert-kaldi-{{ version }}.tar.gz
+  sha256: 82f713d3f974d3b1530fa7091ecf03065549a816c2c6b5e06a2cb388ff6385dd
+  patches:
+    - setup.patch
+    - kaldi-blas.patch
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  entry_points:
+    - write-table-to-pickle = pydrobert.kaldi.command_line:write_table_to_pickle
+    - write-pickle-to-table = pydrobert.kaldi.command_line:write_pickle_to_table
+    - compute-error-rate = pydrobert.kaldi.command_line:compute_error_rate
+    - normalize-feat-lens = pydrobert.kaldi.command_line:normalize_feat_lens
+    - write-table-to-torch-dir = pydrobert.kaldi.command_line:write_table_to_torch_dir
+    - write-torch-dir-to-table = pydrobert.kaldi.command_line:write_torch_dir_to_table
+  script_env:
+    - LAPACKEROOT={{ PREFIX }}  # [not win]
+    - LAPACKEROOT={{ LIBRARY_PREFIX }}  # [win]
+    - LIBPATH={{ LIBRARY_BIN }}  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - numpy
+    # conda-forge implicitly chooses netlib version
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
```

```suggestion
    - libcblas
    - liblapack
```
Please review the links/includes of BLAS in your package. The link checker indicates that some or all of these packages [are not being linked dynamically or are unused](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=498042&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=7012) depending on platform. On conda-forge we require dynamically linking to shared libraries if they are used.

Some libraries are header-only, but I don't think that's the case for BLAS.

---

Thanks for the review!

I pulled the explicit link to libblas but I ended up ignoring the run_exports from liblapacke rather than removing it completely.

Kaldi's source includes "lapacke.h" because it requires a C interface to lapack, but it doesn't look like it uses any of the symbols specific to "lapacke.so".

Is there a better way to do this?

---

If the source includes the header from `lapacke`, but doesn't actually link against `lapacke.so` for whatever reason, then ignoring the run_exports is the way to do it.